specification/quantum/resource-manager/Microsoft.Quantum/preview/2023-11-13-preview/quantum.json

```diff
@@ -0,0 +1,1117 @@
+      "post": {
+        "description": "Get the keys to use with the Quantum APIs. A key is used to authenticate and authorize access to the Quantum REST APIs. Only one key is needed at a time; two are given to provide seamless key regeneration.",
+        "operationId": "Workspace_ListKeys",
+        "x-ms-examples": {
+          "ListKeys": {
+            "$ref": "./examples/ListKeys.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "schema": {
+              "$ref": "#/definitions/ListKeysResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/regenerateKey": {
+      "post": {
+        "description": "Regenerate either the primary or secondary key for use with the Quantum APIs. The old key will stop working immediately.",
+        "operationId": "Workspace_RegenerateKeys",
+        "x-ms-examples": {
+          "RegenerateKey": {
+            "$ref": "./examples/RegenerateKey.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "name": "keySpecification",
+            "in": "body",
+            "required": true,
+            "description": "Which key to regenerate:  primary or secondary.",
+            "schema": {
+              "$ref": "#/definitions/ApiKeySpecification"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Please use 204 response if there is no response body. If you want to use 200 response, it should always have response schema.